### PR TITLE
Reorganizing netobserv release note

### PR DESCRIPTION
**Pull request by skrthomas**
Per @kalexand-rh , this does not need SME review since it is a small reorg change, not a technical change. 

Version(s):

Issue:

I accidentally interrupted a list of release notes with the Network Observability release note, so I needed to reorg it under the list that I interrupted. 
Link to docs preview:

Preview is not working for like 2 hours now, so I screen-captured the local preview I got with asciibinder build. See comment below.
QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by skrthomas** [comment]
Been waiting on preview for 2 hours now and it hasn't shown up. In the spirit of moving this along, I'm pasting a screencapture of my local preview. As you can see, the Network Observability Operator now comes after the list of Enhancements to networking alerts. Previously, it [interrupted](https://docs.openshift.com/container-platform/4.13/release_notes/ocp-4-13-release-notes.html#ocp-4-13-network-observability-1_1-1_2) the list. Preview:
image

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -769,6 +765,10 @@ Netlink messages dropped by OVS vSwitch daemon due to netlink socket buffer over
 Netlink messages dropped by OVS kernel module due to netlink socket buffer overflow. This will result in packet loss.
 ----
 
+[id="ocp-4-13-network-observability-1_1-1_2"]
+==== Network Observability Operator
+The Network Observability Operator releases updates independently from the {product-title} minor version release stream. Updates are available through a single, rolling stream which is supported on all currently supported versions of {product-title} 4. Information regarding new features, enhancements, and bug fixes for the Network Observability Operator can be found in the xref:../networking/network_observability/network-observability-operator-release-notes.adoc[Network Observability release notes].
```
Comment (tmalove) [inline release_notes/ocp-4-13-release-notes.adoc:770]:
```suggestion
The Network Observability Operator releases updates independently from the {product-title} minor version release stream. Updates are available through a single, rolling stream, which is supported on all currently supported versions of {product-title} 4. Information regarding new features, enhancements, and bug fixes for the Network Observability Operator can be found in the xref:../networking/network_observability/network-observability-operator-release-notes.adoc[Network Observability release notes].
```
I think this is a non-restrictive clause. If so, see the ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-with-nonrestrictive-clauses-and-phrases

**Pull request merged**